.gitignore

```diff
@@ -58,3 +58,5 @@ xrootd.spec
 dist
 *.egg-info
 bindings/python/VERSION
+cmake-*
+.idea/
```

Comment:
Hi Andy,

No problem :)

Because I use CLion as IDE with local code deployed on a VM. The cmake-* directories is the equivalent of the `build` directory.
The .idea directory is created by CLion.
I just do not want these directories to be pushed to the project :)

Now that you said this, it looks like I can put these excludes in my local `.git/info/exclude` file. I will therefore remove these two lines from the .gitignore file